app/workers/csv_export_worker.rb

```diff
@@ -0,0 +1,57 @@
+require 'fog/aws'
+
+class CsvExportWorker
+  include Sidekiq::Worker
+  sidekiq_options retry: 0
+  sidekiq_options queue: 'export_csv'
+
+  def perform(search_params, recipient_address)
+    search_params = search_params.symbolize_keys
+    presenter = get_csv_presenter(search_params)
+
+    basename = presenter.filename
+    csv_string = presenter.csv_rows
+
+    file_url = upload_to_s3("#{basename}.csv", csv_string)
+
+    # Send email with link
+    ContentCsvMailer.content_csv_email(recipient_address, file_url).deliver_now
+  end
+
+  def get_csv_presenter(search_params)
+    document_types = FetchDocumentTypes.call[:document_types]
+    organisations = FetchOrganisations.call[:organisations]
+
+    content_items = FindContent.enum(search_params)
```

Comment:
Not for this PR, but something that I would address shortly is:

1. Add models `Organisation` and `DocumentType`
2. Add methods to return all organisations and document types with memoization
3. Remove those parameters from the presenter API

This would simplify how we build the presenter.